### ISSUE TITLE
Merge / autoload go-guru

### ISSUE THREAD

**Issue by the42** [dominikh/go-mode.el#228]
This is a suggestion / enhancement request.

I installed both go-mode as well as go-guru using MELPA. However, when I open a go file, go-mode is enabled yet I first have to issue any guru-command before eg. the key bindings or the Guru menu becomes available.

Therefore I suggest to streamline the go development experience and load guru by default or blend guru into go-mode altogether

**Comment by jsung** [comment]
I believe you just need

    (require 'go-guru)

in your ~/.emacs or ~/emacs.d/init.el (see go-guru-readme.txt).

**Comment by the42** [comment]
I know, what I am suggesting is to auto-enable it instead of having to manually edit `init.el` (sane defaults)

**Comment by dominikh** [comment]
Yeah, we'll do that eventually. I have some changes for go-mode planned, in a couple months, which would merge all the different packages into a single, integrated experience.

**Comment by muirrn** [comment]
Thank you for the suggestion. Guru doesn't work with modules and will soon be superseded by gopls. Once gopls is ready, go-mode will integrate with gopls/lsp-mode in a similar fashion. I'm closing this issue because there is an easy workaround and guru will not be a useful tool going forward.
